graspologic/embed/omni.py

```diff
@@ -164,6 +165,10 @@ def fit(self, graphs, y=None):
         self : object
             Returns an instance of self.
         """
+        if any([isspmatrix_csr(g) for g in graphs]):
+            msg = "OmnibusEmbed does not support scipy.sparse.csr_matrix inputs"
```

Comment:
I know we've mentioned this but it might be worth creating an issue around csr omni because I do not think that that issue exists at the moment.

Reply:
#621